portfolio/src/main/webapp/index.html

```diff
@@ -34,11 +34,90 @@ <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></
                 <p>
                   <strong>A little ABOUT ME! </strong> <small> @bksundar@google.com </small> <br>
                   Hi there! I'm Bhargavi. I was born in LA, and moved to India when I was two years old. I grew up entirely in Chennai, Tamilnadu, and did my schooling in PSBB Nungambakkam. I moved to the US in 2018 to pursue my undergraduate degree in Computer Science. Currently, I am a rising junior at the University of California, Santa cruz.
-                  </p>
+                </p>
               </div>
             </div>
           </article>
         </div>
       </section>
+
+      <section class = "section">
+        <div class="container">
+          <div class = "columns"> 
+            
+            <div class= "column">
+              <!-- This is column 1 content --> 
+			        <article class="message is-info">
```

Comment:
Did you copy this part of code from somewhere? The indentation is broken in index.html, but other files seem to be fine.